README.rst

```diff
@@ -136,6 +136,34 @@ This example shows how to report the traces to Stackdriver Trace:
         project_id='your_cloud_project')
     tracer = tracer_module.Tracer(exporter=exporter)
 
+StackdriverExporter requires the google-cloud-trace package. Install
+google-cloud-trace using `pip`_ or `pipenv`_:
+
+::
+
+    pip install google-cloud-trace
+    pipenv install google-cloud-trace
+
+.. _pip: https://pip.pypa.io
```

Comment:
This `pip` and `pipenv` target are defined at the beginning, please fix this in order to make the CI green.
```
nox > python setup.py check --restructuredtext --strict
running check
warning: check: Duplicate explicit target name: "pip".

warning: check: Duplicate explicit target name: "pipenv".

error: Please correct your package.
```